src/riot-rs-embassy/src/network.rs

```diff
@@ -1,17 +1,30 @@
-//! To provide a custom network configuration, use the `riot_rs::config` attribute macro.
+//! Provides network access.
+//!
+//! The network link to use is selected through Cargo features.
+//! Additionally, the [`riot_rs::config`](riot_rs_macros::config) attribute macro allows to provide
+//! custom network configuration.
+
+#![deny(missing_docs)]
 
 use embassy_net::{Runner, Stack};
 use embassy_sync::once_lock::OnceLock;
 
 use crate::{sendcell::SendCell, NetworkDevice};
 
+/// [MTU](https://en.wikipedia.org/wiki/Maximum_transmission_unit) supported by Ethernet, in bytes.
```

Comment:
Is this accurate? https://en.wikipedia.org/wiki/Maximum_transmission_unit#Applicability says it's 1500 or 1518, depending on whether you count overhead.